### PR TITLE
[FIX] fix condition fulfillment

### PR DESCRIPTION
**Pull request by sentientwaffle**
https://github.com/interledger/five-bells-ledger/pull/133 broke the trader, this should make the compatible again.

### CODE REVIEW

File: src/lib/ledgers/five-bells-ledger.js [hunk 1]
```diff
@@ -38,33 +39,48 @@ FiveBellsLedger.prototype.getState = function (transfer) {
...
+    method: 'put',
+    uri: transfer.id,
+    body: transfer
+  })
+  updateTransfer(transfer, updatedTransfer)
+}
+
+FiveBellsLedger.prototype.putTransferFulfillment = function * (transfer, execution_condition_fulfillment) {
+  const updatedTransfer = yield this._request({
+    method: 'put',
```
Comment (MatthewPhinney) [inline src/lib/ledgers/five-bells-ledger.js]: This should be POST.

File: src/lib/executeSourceTransfers.js [hunk 2]
```diff
@@ -66,7 +66,10 @@ function * executeSourceTransfers (source_transfers, destination_transfers) {
 
   for (let sourceTransfer of source_transfers) {
     log.debug('requesting fulfillment of source transfer')
-    yield ledgers.putTransfer(sourceTransfer)
+    const transferFulfillment = sourceTransfer.execution_condition_fulfillment
```
Comment (MatthewPhinney) [inline src/lib/executeSourceTransfers.js]: Is there a separate case for `cancellation_condition_fulfillment`?

File: src/lib/ledgers/five-bells-ledger.js (hunk 1 above)
Comment (clark800) [inline src/lib/ledgers/five-bells-ledger.js]: Isn't this POST now?

Comment (MatthewPhinney) [inline src/lib/ledgers/five-bells-ledger.js]: Discussed with Stefan and we think it is better to use PUT for this. I have a ticket to update the ledger.

File: src/lib/executeSourceTransfers.js (hunk 2 above)
Comment (sentientwaffle) [inline src/lib/executeSourceTransfers.js]: This is for universal mode, so the cancellation condition is not applicable.

**Comment by MatthewPhinney** [comment]
As discussed, we're going to keep fulfillment a put. So this LGTM.

**Pull request merged**